terra/asset.json

```diff
@@ -1133,6 +1133,7 @@
     "decimals": "6",
     "type": "cw20",
     "circ_supply_api": "https://raw.githubusercontent.com/lioncubdao/logo/main/blue_supply",
+    "total_supply_api": "fromStaticAmount",
```

Comment:
what's the total supply amount for BLUE?

Reply:
Thank you! the token supply of CUB is one trillion (1000000000000)